src/patternfly/components/Page/page.scss

```diff
@@ -407,17 +411,53 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
       }
 
       &.pf-m-no-padding#{$breakpoint-name} {
-        padding: 0;
+        --pf-c-page__main-section--PaddingTop: 0;
+        --pf-c-page__main-section--PaddingRight: 0;
+        --pf-c-page__main-section--PaddingBottom: 0;
+        --pf-c-page__main-section--PaddingLeft: 0;
       }
     }
   }
 }
 
 .pf-c-page__main-wizard {
   flex: 1 0 0;
+  background-color: var(--pf-c-page__main-wizard--BackgroundColor);
   border-top: var(--pf-c-page__main-wizard--BorderTopWidth) solid var(--pf-c-page__main-wizard--BorderTopColor);
 }
 
+.pf-c-page__main-nav,
```

Comment:
what if there was a container called pf-c-page__limit-width that had no styles, but you could put any content inside of it and it would limit the width? e.g. what does someone do if they want to use a container that isnt `--nav`, `--breadcrumb`, `--section` or `--wizard` ? but they want to limit the width? do they have to use one of these containers and then remove the styles ? i guess pro to this would be flexibility in having to maintain this... con: people could limit the width of anything